contrib/k8s/dex-overlord.yaml

```diff
@@ -67,10 +81,18 @@ kind: ConfigMap
 metadata:
   name: dex-connectors
 data:
-  local-connector.json: |
+  connector.json: |
     [
       {
-        "type": "local",
-        "id": "local"
+        "id": "local",
+        "type": "local"
       }
+      # {
```

Comment:
Unfortunately connectors are json not yaml so this comment breaks the deployment.

```
$ kubectl exec $DEX_OVERLORD_POD -- /opt/dex/bin/dexctl --db-url='postgres://postgres@dex-postgres:5432/postgres?sslmode=disable' set-connector-configs '/etc/dex-connectors/connector.json'
Unable to decode connector configs: invalid character '#' after array element
error: error executing remote command: Error executing command in container: Error executing in Docker Container: 1
```